packages/terra-signature/CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ Changelog
 
 Unreleased
 ----------
+### Changed
+* Removing event listeners from the right element during component unmount
+* Fix onChange, it was returning the last segment instead of the array of the whole last line segments
```

Comment:
I don't know how difficult this would be to test, but it feels like this change should have broken one or had a test written to make sure the API stays passive

Reply:
This would technically be a non-passive change if it wasn't considered a bug